File: SistemaAlumnos/bluej/Estudiante.java
public class Estudiante extends Persona {
    private String carrera;

    public Estudiante(String codigo, String nombre, String carrera) {
        super(codigo, nombre);
        this.carrera = carrera;
    }

    @Override
    public void mostrarInformacion() {
        System.out.println(codigo + " - " + nombre + " - " + carrera);
    }
}

File: SistemaAlumnos/SistemaAlumnos/Estudiante.cs
namespace SistemaAlumnos.Logica;

public class Estudiante : Persona
{
    public Estudiante(string codigo, string nombre, string carrera)
        : base(codigo, nombre)
    {
        Carrera = carrera;
    }

    public string Carrera { get; }

    public override void MostrarInformacion()
    {
        Console.WriteLine($"{Codigo} - {Nombre} - {Carrera}");
    }
}
